Add image ordering

private-jobe opened the pull request:
Add image ordering

private-jobe committed:
Add image ordering

bit9labs commented:
I'm not sure when the input changes you wouldn't need to call `refresh_images`. This does a full dump of the database and scrapes the file system. I view this as a very costly function, and if you're only changing the order of display, you wouldn't want to call that. Welcome to hear your feedback

private-jobe committed:
Optimize image sorting

Change from using refresh_image to using get_page as refresh_images
rebuild database.

private-jobe commented:
Didn't actually reflect on that. Copied what I thought was appropriate code from similar function. Corrected.

private-jobe committed:
Remove setting first page on image order change
